README.md

```diff
@@ -29,12 +29,24 @@ TBD... _product statement_
 TBD... _stakeholders and needs_
 
 ## How Rultor Works?
+Once Rultor finds a [merge command](http://doc.rultor.com/basics.html) 
+in one of your GitHub pull requests, it does exactly this:
 
-TBD... _actors and features_
+1. Reads the [.rultor.yml](http://doc.rultor.com/reference.html) 
+ YAML configuration file from the root directory of your repository.
+2. Gets automated build execution command from it, for example `bundle test`.
+3. Checks out your repository into a temporary directory on one of its servers.
+4. Merges pull request into `master` branch.
+5. Starts a new Docker container and runs `bundle test` in it.
```

Comment:
This is wrong and may confuse newbies. In **2** `bundle test` is an example, **5** however states that it **does** run `bundle test` which is not true.

It runs whatever .rultor.yml tells it to run, and this should be clearly stated in number 5.